treeherder/log_parser/parsers.py

```diff
@@ -422,6 +426,15 @@ def add(self, line, lineno):
 
     def parse_line(self, line, lineno):
         """Check a single line for an error.  Keeps track of the linenumber"""
+        # First line of TaskCluster logs almost certainly has this.
+        if line.startswith('[taskcluster '):
+            self.is_taskcluster = True
+
+        # For performance reasons, only do this if we have identified as
+        # a TC task.
```

Comment:
No. Previous experience parsing logs at scale tells me to avoid extra regexp checks at all costs, however. Since this is literally every line of every log, I didn't want to take the chance.